src/components/ACLModal/ACLModal.tsx

```diff
@@ -16,23 +16,20 @@
  */
 
 import * as React from "react";
-import {_} from "translate";
-import {post, get, del} from "requests";
-import {openModal, Modal} from "Modal";
-import {Player} from "Player";
-import {PlayerAutocomplete, PlayerAutocompleteRef} from "PlayerAutocomplete";
-import {GroupAutocomplete} from "GroupAutocomplete";
-import {errorAlerter, rulesText} from "misc";
-
-interface Events {
-}
+import { _ } from "translate";
+import { post, get, del } from "requests";
+import { openModal, Modal } from "Modal";
+import { Player } from "Player";
+import { PlayerAutocomplete, PlayerAutocompleteRef } from "PlayerAutocomplete";
+import { GroupAutocomplete } from "GroupAutocomplete";
+import { errorAlerter, rulesText } from "misc";
+
+interface Events {}
 
 type ACLModalProperties =
-    { game_id: number }
+    | { game_id: number }
```

Comment:
This one is *weird*. but WAI: https://github.com/prettier/prettier/issues/6299

Reply:
I've been learning all kinds of new syntax and things allowed from y'all recently. I personally like this since it's visually consistent.